composer.json

```diff
@@ -5,13 +5,13 @@
   "license": "MIT",
   "minimum-stability": "dev",
   "require": {
-    "alby/php": "dev-main",
+    "getalby/nwc-php": "dev-main",
     "octan/lnd-rest": "dev-master",
     "octan/lnbits": "dev-master",
     "octan/lndhub": "dev-main",
     "octan/btcpay": "dev-main",
-    "firebase/php-jwt": "dev-main",
-    "jorijn/bitcoin-bolt11": "^0.1.0",
+    "firebase/php-jwt": "6.10.0",
+    "getalby/bitcoin-bolt11": "dev-master",
```

Comment:
why did we need to fork this?

Reply:
Thanks! It's not needed now, but we did when we planned to use the nostr-php library. Removing!